includes/bootstrap/SWSDrupalContext.php

```diff
@@ -75,11 +75,15 @@ public function login() {
     // find the Local User Login link - it's only findable in the browser, with Javascript
     // See Behat\Mink\Element\TraversableElement::findLink
     $localuserlogin = $element->findLink("Local User Login");
+    // If we can find a link, that means we're using Javascript.
     if (!is_null($localuserlogin)) {
       // click on the Local User Login link to expose the user name and password fields
       // See Behat\Mink\Element\TraversableElement::clickLink
       $element->clickLink("Local User Login");
+      // Since we're using Javascript, we can use wait().
+      $this->getSession()->wait(5000, '(typeof(jQuery)=="undefined" || (0 === jQuery.active && 0 === jQuery(\':animated\').length))');
     }
+    // Shamelessly stolen from iWaitForAjaxToFinish().
```

Comment:
This comment can go but this looks more like the code we are looking for. 
I wonder if we have a check that is more concrete than the $localuserlogin.